.commit-lint.config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Pay attention to merge request semantic. It is not about proposal, it is about automation of concrete rule we have already accepted as proposal. So this merge request should be for a usual task.

Reply:
I haven't found issues and for the first 5 minutes, I thought that I proposed only commitlint library. But after 10 minutes I've understood that it will be great to create own solution :D